HousingRepairsSchedulingApi/Gateways/DrsAppointmentGateway.cs

```diff
@@ -33,11 +33,25 @@ public class DrsAppointmentGateway : IAppointmentsGateway
         }
 
         public async Task<IEnumerable<AppointmentSlot>> GetAvailableAppointments(string sorCode, string locationId,
-            DateTime? fromDate = null)
+            DateTime? fromDate = null, IEnumerable<AppointmentSlotTimeSpan> allowedAppointmentSlots = default)
         {
             Guard.Against.NullOrWhiteSpace(sorCode, nameof(sorCode));
             Guard.Against.NullOrWhiteSpace(locationId, nameof(locationId));
 
+            var desiredAppointmentSlots = allowedAppointmentSlots;
+
+            if (allowedAppointmentSlots != null)
+            {
+                var dayLightSavingsTimeAdjustedDesiredAppointmentSlots = allowedAppointmentSlots.Select(x =>
+                    new AppointmentSlotTimeSpan
+                    {
+                        StartTime = x.StartTime.Add(TimeSpan.FromHours(-1)),
```

Comment:
Is this a correction for BST to UCT? It might be that I can't see it but will this be correct when we change back to UCT when BST ends?

Reply:
Yes, it's a correction across timezones.
I tried to match the existing behaviour of the system which specified appointment time slots (i.e. 8am to 12pm) and then also a slot 1 hour prior (i.e. 7am to 11am).

This is to address an issue of when the timezones of our system and DRS do not match, i.e. during the period of BST.
DRS has a timezone of UTC, which would match GMT during the period of GMT, but has caused issues during the period of BST.